app/components/public/add-to-calender.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@glimmer/component';
+import moment from 'moment';
+
+export default class AddToCalender extends Component {
+  params = this.args.event;
+
+  get timezone() {
+    return `GMT${moment.tz("Asia/Kolkata").format('Z')}`;  
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403935)